frontend/pages/workspaces/[workspaceId]/members.tsx

```diff
@@ -29,12 +32,22 @@ const ContentWrapper = styled.div`
   display: flex;
 `;
 
+const CountSentence = styled.p`
+  margin: 0;
+  padding-top: 16px;
+`;
+
+const nameCell: FC<User> = ({ name }) => <div>{name}</div>;
+const emailAddressCell: FC<User> = ({ emailAddress }) => (
+  <a href={`mailto:${emailAddress}`}>{emailAddress}</a>
```

Comment:
Do we need to escape `emailAddress` here, maybe with `encodeURIComponent`? I'm not entirely sure, to be honest. It can't break out of the `href` attribute, because React takes care of that. But would an email address like "jan\\?cc=foo@example.com" be possible?

Reply:
According to https://tools.ietf.org/html/rfc6068 the email address should be percent encoded:

> A number of characters that can appear in <addr-spec> MUST be percent-encoded.  These are the characters that cannot appear in a URI according to [STD66] as well as "%" (because it is used for percent-encoding) and all the characters in gen-delims except "@" and ":" (i.e., "/", "?", "#", "[", and "]").  Of the characters in sub-delims, at least the following also have to be percent- encoded: "&", ";", and "=".

I _think_ that encodeURIComponent should be fine, then.

```suggestion
  <a href={`mailto:${encodeURIComponent(emailAddress)}`}>{emailAddress}</a>
```

Reply:
React automatically encodes everything inside <href>

Reply:
Nevermind. `encodeURIComponent` encodes the "@" sign, which is not what we want. `encodeURI` could work.